.github/workflows/gradle.yml

```diff
@@ -25,7 +25,7 @@ jobs:
     - name: create UI distribution
       run: gradle distZip
     - name: Archive jars 
-      uses: actions/upload-artifact@v2
+      uses: actions/upload-artifact@v4
```

Comment:
Use the full version name or sha hash for actions.

Reply:
Thanks, updated to use full version name